dashboard.php

```diff
@@ -5,155 +5,67 @@
-                    </div>
-                    <div class="job-secondary">
-                        <div class="job-edit">
-                            <a href="#">View Submissions</a>
-                            <a href="#">Edit</a>
-                        </div>
-                    </div>
-                </li>
-                <li class="job-card">
-                    <div class="job-primary">
-                        <h2 class="job-title"><a href="#">Front End Developer</a></h2>
-                        <div class="job-meta">
-                            <a class="meta-company" href="#">Company Awesome Ltd.</a>
-                            <span class="meta-date">Posted 14 days ago</span>
-                        </div>
-                        <div class="job-details">
-                            <span class="job-location">The Hague (The Netherlands)</span>
-                            <span class="job-type">Contract staff</span>
-                        </div>
-                    </div>
-                    <div class="job-secondary">
-                        <div class="job-edit">
-                            <a href="#">View Submissions</a>
-                            <a href="#">Edit</a>
-                        </div>
-                    </div>
-                </li>
-                <li class="job-card">
-                    <div class="job-primary">
-                        <h2 class="job-title"><a href="#">Front End Developer</a></h2>
-                        <div class="job-meta">
-                            <a class="meta-company" href="#">Company Awesome Ltd.</a>
-                            <span class="meta-date">Posted 14 days ago</span>
-                        </div>
-                        <div class="job-details">
-                            <span class="job-location">The Hague (The Netherlands)</span>
-                            <span class="job-type">Contract staff</span>
-                        </div>
-                    </div>
-                    <div class="job-secondary">
-                        <div class="job-edit">
-                            <a href="#">View Submissions</a>
-                            <a href="#">Edit</a>
-                        </div>
-                    </div>
-                </li>
-            </ul>
-            <div class="jobs-pagination-wrapper">
-                <div class="nav-links">
-                    <a class="page-numbers current">1</a>
-                    <a class="page-numbers">2</a>
-                    <a class="page-numbers">3</a>
-                    <a class="page-numbers">4</a>
-                    <a class="page-numbers">5</a>
                 </div>
             </div>
         </div>
-    </section>
+		<?php require_once( $_SERVER['DOCUMENT_ROOT'] . '/includes/theme-compat/jobs-listing.php' );
+
+		$s_parameter = '';
+		if ( ! empty( $search ) ) {
+			$s_parameter = '?search=' . $search;
+		}
+		$page_url    = 'dashboard.php' . $s_parameter;
```

Comment:
```suggestion
                $page_url    = 'dashboard.php' 
		if ( ! empty( $search ) ) {
			$page_url    = 'dashboard.php' . '?search=' . $search;
		}
		
```

Or if you would like to make it even better, I suggest looking into this: https://stackoverflow.com/questions/5809774/manipulate-a-url-string-by-adding-get-parameters